pages/common/v.md

```diff
@@ -0,0 +1,36 @@
+# v
+
+> A tool for managing V source code.
+> More information: <https://docs.vlang.io/getting-started.html>.
+
+- Compile a file and output its executable:
+
+`v {{/path/to/file.v}}`
+
+- Compile the current project or directory:
+
+`v .`
+
+- Compile a file and [run] it:
+
+`v run {{path/to/file.v}}`
+
+- Compile a file and [run] it, output its executable:
+
+`v crun {{path/to/file.v}}`
+
+- Re-compile on every modification to a file:
+
+`v watch {{path/to/file.v}}`
+
+- Re-[run] on every modification to a file:
```

Comment:
```suggestion
- Compile a file and run it:

`v run {{path/to/file.v}}`

- Compile a file and run it, output its executable:

`v crun {{path/to/file.v}}`

- Re-compile on every modification to a file:

`v watch {{path/to/file.v}}`

- Re-run on every modification to a file:
```

Usually, we only use mnemonics when the short form of an argument appears in the example.